docs/source/platform/operation-registry.mdx

```diff
@@ -169,6 +173,46 @@ const server = new ApolloServer({
 });
 ```
 
+<ExpansionPanel title="Running Federation using the Apollo Gateway? These same steps should work!">
+
+The full setup:
+
+```js{21-25}
+const { ApolloServer } = require('apollo-server');
+const { ApolloGateway } = require("@apollo/gateway");
+
+const gateway = new ApolloGateway({
+  serviceList: [
+    { name: 'accounts', url: 'http://localhost:4001' },
+    // more services
+  ],
+});
+
+(async () => {
+  const { schema, executor } = await gateway.load();
+
+  const server = new ApolloServer({
+    // Existing configuration
+    schema,
+    executor,
+    subscriptions: false
+    // ...
+    // New configuration
+    plugins: [
+      require("apollo-server-plugin-operation-registry")({
+        forbidUnregisteredOperations: true
+      })
+    ]
+  });
+
+  server.listen().then(({ url }) => {
+    console.log(`🚀 Server ready at ${url}`);
+  });
+})();
+```
+
+</ExpansionPanel>
+
 ####5.1 Optionally, set the schema tag
```

Comment:
There should be a space between the last `#` and the number `5` for the heading to render correctly